chromeipass/options/options.html

```diff
@@ -419,7 +433,7 @@ <h3 id="myModalLabel">Remove specified credential fields?</h3>
 			<hr>
 
 			<div class="footer">
-				<p>2013 - Perry Nguyen, Lukas Schulze</p>
+				<p>2010-2016 - Perry Nguyen, Lukas Schulze</p>
```

Comment:
Lets drop this line altogether as part of your fixes, since people have started to submit PR, I would like a more general approach to recognize everyone.